data/json/overmap/overmap_terrain/overmap_terrain_military.json

```diff
@@ -215,15 +215,6 @@
     "see_cost": 2,
     "flags": [ "KNOWN_DOWN", "RISK_HIGH", "SOURCE_WEAPON", "SOURCE_AMMO" ]
   },
-  {
-    "type": "overmap_terrain",
-    "id": "bunker_basement",
```

Comment:
Overmap terrain savegame migrations also needs to be added (e.g. like in https://github.com/CleverRaven/Cataclysm-DDA/pull/53447).